atlp-rwanda/cabal-phantom-backend
# #8yVIZ5wE Driver and operator able to update their profile

###  What does this PR do?
This PR requests to merge ft-update-profile-zqKf1dcP into develop

###  Description of Task to be completed?
The operator and driver should be able to update information about their own profile.
The operator and driver profile should be able accessible publicly.
Appropriate error messages are displayed on unsuccessful attempts.

###  How should this be manually tested?

1. Clone repo if you haven't then do git checkout origin/ft-update-profile-zqKf1dcP
      -install all packages
      -make sure your db is connected with the help of .env.example file
      -log in your email using help of .env.example file email 

2. POST http://localhost:5000/api/v1/auth/login login using these to have permission :email: **niyiragiragerard@gmail.com** and password:**8KF9hfF$XiKU** 
3. GET one user using their id at http://localhost:5000/api/v1/auth/profiles/{id}
4.PUT http://localhost:5000/api/v1/auth/updateProfile to update yourself and use **Bearer <token> in Authorization Headers**

###  What are the relevant pivotal tracker stories?
[zqKf1dcP](https://trello.com/c/zqKf1dcP/38-update-profile)

###  Screenshots (if appropriate)
<img width="1440" alt="Screen Shot 2020-10-23 at 08 19 42" src="https://user-images.githubusercontent.com/68019992/97668332-5e9ba600-1a8a-11eb-8ede-e9c0b957d82e.png">
